dts/x86/ia32.dtsi

```diff
@@ -0,0 +1,51 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "intel,ia32";
```

Comment:
this seems wrong, what specific cpu are we using with qemu?

Reply:
According to boards/x86/qemu_x86/Makefile.board, we are passing "-cpu qemu32" to the emulator.
"ia32" is the name of the SOC target in Zephyr, which is a generic pc-like platform. I don't have an opinion whether this needs to be rearranged.